### ISSUE TITLE
Git-Fusion image lacks package installation in Dockerfile

### ISSUE THREAD

**Issue by adamczykjac** [ambakshi/docker-perforce#3]
I ran into several issues while trying to run Git-Fusion image

1. Git-Fusion is based on perforce-server image, which version `2015.2` dependencies do not seem to exist in yum repos anymore
2. Git-Fusion version pinpointed - 2015.3 - and the packages installed in Dockerfile do not provide the `configure-git-fusion.sh` file. The only package, which provides that file - `helix-git-fusion-base` - is available in 2016.1 version of `helix-git-fusion`, so there is a need to update the `FUSION_VERSION` env variable as well.
3. While finally running configure-git-fusion.sh there are obsolete parameters passed in etc like `--p4root`,  and later on further error pops out with
`Unable to trust the server ssl:1666`

**Comment by bicschneider** [comment]
Similar to @2: 
```
Resolving Dependencies
--> Running transaction check
---> Package helix-p4d.x86_64 0:2015.2-1377629 will be installed
--> Processing Dependency: helix-p4dctl = 2015.2 for package: helix-p4d-2015.2-1377629.x86_64
--> Processing Dependency: helix-cli = 2015.2 for package: helix-p4d-2015.2-1377629.x86_64
--> Processing Dependency: helix-p4d-base = 2015.2 for package: helix-p4d-2015.2-1377629.x86_64
---> Package helix-p4dctl.x86_64 0:2016.1-1374211 will be installed
--> Running transaction check
---> Package helix-p4d.x86_64 0:2015.2-1377629 will be installed
--> Processing Dependency: helix-p4dctl = 2015.2 for package: helix-p4d-2015.2-1377629.x86_64
--> Processing Dependency: helix-cli = 2015.2 for package: helix-p4d-2015.2-1377629.x86_64
---> Package helix-p4d-base.x86_64 0:2015.2-1377629 will be installed
--> Finished Dependency Resolution
 You could try using --skip-broken to work around the problem
Error: Package: helix-p4d-2015.2-1377629.x86_64 (perforce)
           Requires: helix-cli = 2015.2
           Installed: helix-cli-2016.1-1374211.x86_64 (@perforce)
               helix-cli = 2016.1-1374211
           Available: helix-cli-2015.2-1377629.x86_64 (perforce)
               helix-cli = 2015.2-1377629
Error: Package: helix-p4d-2015.2-1377629.x86_64 (perforce)
           Requires: helix-p4dctl = 2015.2
           Available: helix-p4dctl-2015.2-1377629.x86_64 (perforce)
               helix-p4dctl = 2015.2-1377629
           Installing: helix-p4dctl-2016.1-1374211.x86_64 (perforce)
               helix-p4dctl = 2016.1-1374211
 You could try running: rpm -Va --nofiles --nodigest
The command '/bin/sh -c yum install -y helix-p4d-$P4D_VERSION helix-p4dctl' returned a non-zero code: 1
make: *** [perforce-server] Error 1
```

**Comment by ambakshi** [comment]
I'll look into this today, thanks for the report.
